### PR TITLE
Add script to check that Policy Manager provides data consent prompt …

### PR DESCRIPTION
**Pull request by vvvakulenko**
…from the policy table upon request from HMI

### CODE REVIEW

**Comment by vvvakulenko** [comment]
@maria-mih Please review

File: test_scripts/Polices/Related_HMI_API/ATF_PolManager_provides_data_consent_prompt_on_HMI_request.lua [hunk 1]
```diff
@@ -0,0 +1,82 @@
+--UNREADY
+-- shiuld be modified
```
Comment (maria-mih) [inline test_scripts/Polices/Related_HMI_API/ATF_PolManager_provides_data_consent_prompt_on_HMI_request.lua]: Please add a short description what should be modified.

File: test_scripts/Polices/Related_HMI_API/ATF_PolManager_provides_data_consent_prompt_on_HMI_request.lua [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+testCasesForBuildingSDLPolicyFlag:Update_PolicyFlag("EXTENDED_POLICY", "EXTERNAL_PROPRIETARY")
+testCasesForBuildingSDLPolicyFlag:CheckPolicyFlagAfterBuild("EXTENDED_POLICY","EXTERNAL_PROPRIETARY")
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('mobile_session')
+
+--[[ Preconditions ]]
```
Comment (maria-mih) [inline test_scripts/Polices/Related_HMI_API/ATF_PolManager_provides_data_consent_prompt_on_HMI_request.lua]: If you do not have any preconditions please remove this section

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**